db/migrate/20210429121634_create_oidc_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOidcUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :oidc_users do |t|
+      t.string :sub, null: false
```

Comment:
what does `:sub` refer to, in this case?
Is there a more descriptive name we could use?

Reply:
It's the subject identifier, also used here: https://github.com/alphagov/account-api/blob/a53a71fea170d3497edc51b2c20f8568cff3312e/app/controllers/authentication_controller.rb#L36